src/explorer/model/Dependency.ts

```diff
@@ -50,6 +56,16 @@ export class Dependency extends TreeNode implements ITreeItem {
         return this._scope;
     }
 
+    public get uri(): vscode.Uri {
+        const filePath: string = localPomPath(this._gid, this._aid, this._version);
+        let uri: vscode.Uri = vscode.Uri.file(filePath);
+        uri = uri.with({authority: this.projectPomPath});
```

Comment:
why update `authority`?

Reply:
In multi-module projects, one dependency with the same `localPomPath` may import in different Maven modules. In order to distinguish them (maybe one has conflicts but the other one does not), I use `authority`.

Reply:
you should take notes for any tricks (anti common sense) in comments.